app/experimenter/nimbus-ui/src/index.tsx

```diff
@@ -5,16 +5,22 @@
 import React from "react";
 import ReactDOM from "react-dom";
 import App from "./components/App";
+import AppErrorBoundary from "./components/AppErrorBoundary";
+import config, { readConfig } from "./lib/config";
+import sentryMetrics from "./services/sentry";
 import "./styles/index.scss";
-import { readConfig } from "./lib/config";
 
 try {
+  sentryMetrics.configure(config.sentry_dsn, config.version);
```

Comment:
Oh bah, not critical, but it looks like this resulted in some `console.logs` in the main index test file. Could stub them.